### ISSUE TITLE
consider directing users to apptainer instead of singularity

### ISSUE THREAD

**Issue by kaczmarj** [Cadaei-Yuvxvs/OSHy-X#11]
the community-developed version of singularity was recently renamed to "apptainer" and has joined the linux foundation (see [news release](https://apptainer.org/news/community-announcement-20211130)). i'll be honest, i didn't realize there were two different forks of singularity being developed in parallel -- and this rebranding is an effort to clarify things. so now, apptainer is the community-driven software that is part of the linux foundation, and singularity is the commercially-developed (but still open source) product being developed by sylabs. apptainer has the same command line interface as singularity and also ships with a `singularity` command (for the time being).

long story short, the developers of this repository might consider directing users to apptainer.

this issue is not related to my review of the joss submission.

cc: @stebo85

**Comment by stebo85** [comment]
Agree @kaczmarj - we are currently testing apptainer and it works very well! We will be switching over soon.

**Comment by Cadaei-Yuvxvs** [comment]
Thanks for bringing this to my attention! I'll certainly look into this.

**Comment by Cadaei-Yuvxvs** [comment]
Repository is now updated to reference Apptainer: d4bc623c6f65b58040022aefe64d93a82eb17fed
